data/mods/Aftershock/mobs/mutants.json

```diff
@@ -0,0 +1,70 @@
+[
+  {
+    "id": "mon_mutant_cryo",
+    "type": "MONSTER",
+    "name": { "str": "frost maddened human" },
+    "description": "Sky blue irises cover the entirety of the eye.  What happened to this person during their deep freeze?",
+    "default_faction": "mutant",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 40,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "cold", "amount": 4 } ],
+    "melee_cut": 0,
+    "dodge": 2,
+    "harvest": "human",
+    "vision_day": 30,
+    "vision_night": 3,
+    "death_drops": "cryo_lab_drops",
+    "death_function": [ "NORMAL" ],
+    "zombify_into": "mon_zombie",
```

Comment:
I've been meaning to ask about this, so I might as well.

Should we have this if Aftershock is going into new lore with no actual zombies? or is it just in for now as a placeholder?

Reply:
Good question!

Reply:
Removing zombify :)